javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -737,26 +816,10 @@ public void slang_persistent(Blackhole bh) {
 
     /** Sequential access for all elements */
     public static class Iterate extends Base {
-        @State(Scope.Thread)
```

Comment:
no need to recreate it in `Iterate` as we don't change the `javaMutable` in any way :)